src/platform/languageClientApi.nim

```diff
@@ -0,0 +1,34 @@
+import std/jsffi
+import js/[jsPromise, asyncjs, jsre, jsNode]
+
+export jsffi, jsPromise, jsNode
+
```

Comment:
Could you add a comment with a link to the API docs, that way others can quickly get to the right place if they need to confirm, fix, update, whatever this shim.

Reply:
ok